main.go

```diff
@@ -14,21 +13,30 @@ var log = &logrus.Logger{
 	Out:       os.Stderr,
 	Formatter: new(logrus.TextFormatter),
 	Hooks:     make(logrus.LevelHooks),
-	Level:     logrus.ErrorLevel,
+	Level:     logrus.InfoLevel, // Specifies we only want logs in stdout @ Info or higher
```

Comment:
I think the InfoLevel is self explainatory

Reply:
Updated in [5db1db0](https://github.com/endigma/disgord-starter/commit/34cc0e94049962b1f680bc08e904c01edd55efb8)